### ISSUE TITLE
fillna() got an unexpected keyword argument 'fill_nan'

### ISSUE THREAD

**Issue by laozhuang727** [vaexio/vaex#499]
Hi dear，

Follow the document to process the fillna, but got below error, would you please help to take a look?

x = np.array([3, 1, np.nan, 10, np.nan])
        df = vaex.from_arrays(x=x)
        df_filled = df.fillna(value=-1, column_names=['x'])
        print(df_filled)

ERROR:MainThread:vaex:error evaluating: x at rows 0-5
Traceback (most recent call last):
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/scopes.py", line 94, in evaluate
    result = self[expression]
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/scopes.py", line 141, in __getitem__
    raise KeyError("Unknown variables or column: %r" % (variable,))
KeyError: "Unknown variables or column: 'fillna(__original_x, -1, fill_nan=True, fill_masked=True)'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/dataframe.py", line 3467, in table_part
    values[name] = df.evaluate(name)
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/dataframe.py", line 5038, in evaluate
    dtype = dtypes[expression] = self.dtype(expression, internal=False)
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/dataframe.py", line 2005, in dtype
    data = self.evaluate(expression, 0, 1, filtered=False, internal=True, parallel=False)
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/dataframe.py", line 5143, in evaluate
    value = scope.evaluate(expression)
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/scopes.py", line 94, in evaluate
    result = self[expression]
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/scopes.py", line 136, in __getitem__
    self.values[variable] = self.evaluate(expression)  # , out=self.buffers[variable])
  File "/Users/ryan/opt/miniconda3/envs/tensorflow/lib/python3.6/site-packages/vaex/scopes.py", line 100, in evaluate
    result = eval(expression, expression_namespace, self)
  File "<string>", line 1, in <module>
TypeError: fillna() got an unexpected keyword argument 'fill_nan'

**Comment by JovanVeljanoski** [comment]
Hi @laozhuang727 

Thanks for the report. This is a known issue, we are working on a fix.

In the meantime, you can do this:

```
x = np.array([3, 1, np.nan, 10, np.nan])
df = vaex.from_arrays(x=x)
df['x_filled'] = df.x.fillna(value=-1)
df
```

Let's keep this issue open until the fix is merged. 

I hope this helps!
Jovan.

**Comment by JovanVeljanoski** [comment]
Hi @laozhuang727 

This was fixed by #500 .

Thanks for letting us know!
